packages/constellation/package.json

```diff
@@ -36,29 +37,36 @@
     "@babel/preset-react": "7.14.5",
     "@babel/runtime": "7.16.7",
     "@blockchain-com/icons": "workspace:*",
+    "@luigiminardim/storybook-addon-globals-controls": "^0.1.3",
+    "@mdx-js/react": "^1.6.22",
     "@rollup/plugin-babel": "5.3.0",
     "@rollup/plugin-commonjs": "21.0.1",
     "@rollup/plugin-node-resolve": "13.1.2",
     "@rollup/plugin-typescript": "8.3.0",
-    "@storybook/addon-actions": "6.4.19",
-    "@storybook/addon-docs": "6.4.19",
-    "@storybook/addon-essentials": "6.4.19",
-    "@storybook/addon-links": "6.4.19",
-    "@storybook/react": "6.4.19",
-    "@storybook/storybook-deployer": "2.8.10",
-    "@storybook/theming": "6.4.19",
+    "@storybook/addon-actions": "^6.5.9",
```

Comment:
Could you pin all these version numbers? It's just something we always do for added security.

Reply:
Yep definitely 👍